src/css/_includes/index.scss

```diff
@@ -112,7 +112,7 @@ div.footer-btns {
 .index-search {
   position: relative;
   max-width: 55rem;
-  margin: 3rem auto 1rem;
+  margin: 0 auto;
```

Comment:
@mydea why did we remove the padding on the search bar? It definitely looked much more balanced before no?

![image](https://github.com/getsentry/sentry-docs/assets/1421724/3cc5056c-44a7-4612-8deb-7ebd27450d3f)
vs
![image](https://github.com/getsentry/sentry-docs/assets/1421724/f7db3420-c7ec-4fe3-9ce3-5b3041a73674)